.appveyor.yml

```diff
@@ -60,7 +60,7 @@ after_build:
 - ps:  clcache -z
 #- 7z a bitcoin-%APPVEYOR_BUILD_VERSION%.zip %APPVEYOR_BUILD_FOLDER%\build_msvc\%platform%\%configuration%\*.exe
 test_script:
-- cmd: src\test_bitcoin.exe -k stdout -e stdout 2> NUL
+- cmd: src\test_bitcoin.exe -l test_suite -k stdout -e stdout 2> NUL
```

Comment:
that is going to print the boost test messages always, even if everything passes

I am not sure about any log limits for appveyor, but this seems a bit verbose to me

Reply:
Jup, it does print it unconditionally: https://ci.appveyor.com/project/DrahtBot/bitcoin/builds/29582061#L1000